### PR TITLE
Create ar_messages.dart

### PR DESCRIPTION
**Pull request by ammaratef45**
Added Arabic support for the library.

### CODE REVIEW

File: timeago/lib/src/messages/ar_messages.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import 'package:timeago/src/messages/lookupmessages.dart';
+
+class EnMessages implements LookupMessages {
```
Comment (andresaraujo) [inline timeago/lib/src/messages/ar_messages.dart]: these classes should be named after the locale names, in this case the name should be `ArMessages` and `ArShortMessages`

Comment (ammaratef45) [reply timeago/lib/src/messages/ar_messages.dart]: @andresaraujo oops!
Okay I've edited them

**Comment by andresaraujo** [comment]
Thanks @ammaratef45! you locale is now included in v2.0.12

**Comment by ammaratef45** [comment]
@andresaraujo thanks

**Pull request merged**